code/modules/client/preference/loadout/loadout_general.dm

```diff
@@ -1,3 +1,50 @@
 /datum/gear/dice
-	display_name = "d20"
-	path = /obj/item/weapon/dice/d20
\ No newline at end of file
+	display_name = "a d20"
+	path = /obj/item/weapon/dice/d20
+
+/datum/gear/uplift
+	display_name = "a pack of Uplifts"
+	path = /obj/item/weapon/storage/fancy/cigarettes/cigpack_uplift
+
+/datum/gear/robust
+	display_name = "a pack of Robusts"
+	path = /obj/item/weapon/storage/fancy/cigarettes/cigpack_robust
+
+/datum/gear/carp
+	display_name = "a pack of Carps"
+	path = /obj/item/weapon/storage/fancy/cigarettes/cigpack_carp
+
+/datum/gear/midori
+	display_name = "a pack of Midoris"
+	path = /obj/item/weapon/storage/fancy/cigarettes/cigpack_midori
+
+/datum/gear/lighter
+	display_name = "a cheap lighter"
+	path = /obj/item/weapon/lighter
+
+/datum/gear/cigar
+	cost = 5
+	display_name = "a cigar"
+	path = /obj/item/clothing/mask/cigarette/cigar
+
+/datum/gear/flask
+	cost = 3
+	display_name = "a flask"
+	path = /obj/item/weapon/reagent_containers/food/drinks/flask/barflask
+
+/datum/gear/rock
+	display_name = "a pet rock"
+	path = /obj/item/toy/pet_rock
+
+/datum/gear/pipe
+	display_name = "a smoking pipe"
+	path = /obj/item/clothing/mask/cigarette/pipe
+
+/datum/gear/pai
+	display_name = "a pAI card"
+	path = /obj/item/device/paicard
+
+/datum/gear/sechud
+	display_name = "a classic security HUD"
+	path = /obj/item/clothing/glasses/hud/security
```

Comment:
This is weird, when literally every single one of these jobs spawns in with one.